uint16[]  id_list

float64[] profile_velocity__deg_s
float64[] profile_acceleration__deg_ss